atd-vzv/src/views/map/Map.js

```diff
@@ -99,8 +91,13 @@ const Map = () => {
   // Create ref to map to call Mapbox GL functions on instance
   const mapRef = useRef();
 
+  const { width } = useWindowSize();
+  const { bootstrapMedium } = responsive;
+  const isMobile = width <= bootstrapMedium;
```

Comment:
you should test this in a mobile browser but the chrome simulator had issues with the card width.

Also, the `x` close button wasn't working but a click off the card did.

![Screen Shot 2020-04-28 at 2 06 54 AM](https://user-images.githubusercontent.com/5697474/80457578-28237980-88f5-11ea-911b-781ccf970769.png)

Reply:
Oh nice catch! I'll check out the "x" button in BrowserStack. For the card width, I'm not sure if there is a way to guarantee that the popup always opens 100% visible within the viewport since the same issue is present in the example in the docs. http://visgl.github.io/react-map-gl/examples/controls

Maybe we should brainstorm on this some more? The screenshot below captures the issues with both styles of info boxes in this PR when used on mobile (static or popup). Either controls are covered up by the static info box or the popup is off screen until dragged into center.

<img width="366" alt="Screen Shot 2020-04-28 at 9 52 38 AM" src="https://user-images.githubusercontent.com/37249039/80502264-00a0d100-8936-11ea-9261-4ae8d011c28c.png">

Reply:
I fixed the close button. I was missing a handler for the close button so I added that (`onClose={() => setSelectedFeature(null)}`). This was hard to notice because it turns out the existing close popup behavior was due to a bug in the click handler for the crash point layers.

Basically, the handler sets a `selectedFeature` if a GeoJSON feature exists where a click occurs. If not, it was setting `selectedFeature` as `undefined`. The logic to render the info box depends on a truthy value for `selectedFeature` so a setting a falsey value was "closing" the info box.